blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/EventModuleLoader.java

```diff
@@ -0,0 +1,45 @@
+package com.rackspacecloud.blueflood.utils;
+
+import com.rackspacecloud.blueflood.io.GenericElasticSearchIO;
+import com.rackspacecloud.blueflood.service.Configuration;
+import com.rackspacecloud.blueflood.service.CoreConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class EventModuleLoader {
+    private static final Logger log = LoggerFactory.getLogger(EventModuleLoader.class);
+    private static GenericElasticSearchIO instance = null;
+
+    public static GenericElasticSearchIO getInstance() {
+        return instance;
```

Comment:
TODO: instance is null when you invoke getInstance() statically without invoking loadEventModule() 
It should probably be-
```
if (instance == null) 
{
    instance = loadEventModule();
}
```